example-get-started-experiments/code/.github/workflows/dvc-studio.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       - name: Train
         env:
           REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
-          STUDIO_TOKEN: ${{ secrets.STUDIO_TOKEN }}
+          DVC_STUDIO_TOKEN: ${{ secrets.DVC_STUDIO_TOKEN }}
```

Comment:
We were using a mix of `STUDIO_TOKEN` and `DVC_STUDIO_TOKEN` before. Now it should consistently use `DVC_STUDIO_TOKEN` everywhere.

@tibor-mach Minor, but maybe we should use the env var in the deployment example to match here and because it seems a bit more straightforward for the CI case (no real need to configure it globally for one-time use).